dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jaxwsClient-2.2.feature

```diff
@@ -8,7 +8,7 @@ IBM-Process-Types: client
  com.ibm.websphere.appserver.containerServices-1.0, \
  com.ibm.websphere.appserver.classloading-1.0, \
  com.ibm.websphere.appserver.javax.mail-1.5; ibm.tolerates:="1.6", \
- com.ibm.websphere.appserver.jaxb-2.2
+ com.ibm.websphere.appserver.jaxb-2.2; ibm.tolerates:=2.3
```

Comment:
Should JAX-WS 2.2 be tolerating JAX-B 2.3?  I think we would want to keep JAX-B 2.2 and JAX-WS 2.2 in sync, and then the 2.3's would match up